scripts/common.Makefile

```diff
@@ -53,9 +53,9 @@ MAKE_C := $(MAKE) --no-print-directory --directory
 #
 
 
-.PHONY: help
+.PHONY: hel%
```

Comment:
is this % intended and what does it do?

Reply:
This fixes a warning produced in the catalogs ``help`` Makefile recipe
This is how something like "inheritance" can be achieved with makefiles. Check https://mattandre.ws/2016/05/makefile-inheritance/ for details.